### DISCUSSION TITLE
[GENERAL QUESTION] Use keyboard keys with GP2040-CE

### DISCUSSION THREAD

**Discussion by Alicka6** [OpenStickCommunity/GP2040-CE#341] [Q&A]
I want to turn my old mechanical keyboard into a DIY into a Hitbox Style controller but i dont know if its ok to use the mechanical keys from the keyboard instead of Arcade buttons. if its possible to do. How do i wire it?

**Comment by bsstephan** [comment]
Arcade buttons and keyboard keys are, in the end, the same type of mechanical switch, so you'd wire it the same as you'd wire a pushbutton.

**Comment by TheTrainGoes** [comment]
Hi @Alicka6 ,

This is possible but you would not be able to use the matrix from the keyboard layout.  You would need to isolate the individual keys.
